File: Assets/VirtualCity/ProtoDefine/Java/mall/ReqQueryBillMessage.java
package com.kingston.jforgame.server.game.mall.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.mall.MallDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

@MessageMeta(module = Modules.SHOPPING,cmd = MallDataPool.REQ_QUERYBILL)
public class ReqQueryBillMessage extends Message {
    /**类型*/
    @Protobuf(order = 1)
    private int billType;
    /**最后一条记录的时间*/
    @Protobuf(order = 3)
    private String lastDate;

    public int getBillType() {
        return billType;
    }

    public void setBillType(int billType) {
        this.billType = billType;
    }

    public String getLastDate() {
        return lastDate;
    }

    public void setLastDate(String lastDate) {
        this.lastDate = lastDate;
    }

}

File: Assets/VirtualCity/ProtoDefine/ReqQueryBillMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class ReqQueryBillMessage {
    /**类型*/
[ProtoMember(1)]
    public int billType;
    /**最后一条记录的时间*/
[ProtoMember(2)]
    public string lastDate;

    public int getBillType() {
        return billType;
    }

    public void setBillType(int billType) {
        this.billType = billType;
    }

    public string getLastDate() {
        return lastDate;
    }

    public void setLastDate(string lastDate) {
        this.lastDate = lastDate;
    }

}
}